### PR TITLE
Update logging in CLI commands

### PR DESCRIPTION
**Pull request by smilledge**
### ✨️ Improvements
- Replaced usage of `console` with `logger` in CLI commands
- Logging from CLI commands will now respect the `LOG_LEVEL` env var
log level demo
- Useful information (eg, IDs from the create role/user) is always written to stdout so that output can be piped to other commands
pipe output demo

### CODE REVIEW

File: api/src/cli/commands/init/index.ts [hunk 1]
```diff
@@ -49,19 +48,19 @@ export default async function init(): Promise<void> {
...
-			console.log();
-			console.log('Please try again');
-			console.log();
+			logger.info('');
+			logger.info('Something went wrong while seeding the database:');
+			logger.info('');
+			logger.info(`${chalk.red(`[${err.code || 'Error'}]`)} ${err.message}`);
+			logger.info('');
+			logger.info('Please try again');
+			logger.info('');
```
Comment (rijkvanzanten) [inline api/src/cli/commands/init/index.ts:57]: We can drop the `''` here. With `console.log`, it'll print an empty line, but with `pino`, it'll construct a whole new logging object without a message, which isn't what we'd want.

```suggestion
			logger.info(err, `
Something went wrong while seeding the database:

${chalk.red(`[${err.code || 'Error'}]`)} ${err.message}

Please try again
			`);
```

File: api/src/cli/commands/init/index.ts [hunk 2]
```diff
@@ -71,10 +70,9 @@ export default async function init(): Promise<void> {
 
 	await createEnv(dbClient, credentials!, rootPath);
 
-	console.log();
-	console.log();
-
-	console.log(`Create your first admin user:`);
+	logger.info('');
+	logger.info('');
+	logger.info(`Create your first admin user:`);
```
Comment (rijkvanzanten) [inline api/src/cli/commands/init/index.ts:75]:
```suggestion
	logger.info(`


Create your first admin user:`);
```

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]
I'm wondering if we should keep relying on console.log for `init`, as that's meant to be an interactive questionnaire, and not a flow you'd ever disable the "logs" for 🤔 

The others make sense to me 👍🏻

**Comment by smilledge** [comment]
Yeah definitely, I didn't think of that. We can use stdout for the init command too? 

Output looks like this now;
Screenshot from 2021-08-31 13-45-48

**Comment by rijkvanzanten** [comment]
> We can use stdout for the init command too?

Sure! That works. What's the perk of relying on stdout directly instead of console log? Doesn't console log just print to stdout with a newline character at the end?

**Comment by smilledge** [comment]
> What's the perk of relying on stdout directly instead of console log? Doesn't console log just print to stdout with a newline character at the end?

Yeah there isn't really any different here, just semantics and avoiding an `eslint-disable` 🤷‍♂️

**Review by rijkvanzanten — APPROVED** [review]

**Comment by rijkvanzanten** [comment]
LGTM! Thanks Sam 🙂

**Pull request merged**